azkaban-web-server/src/main/java/cloudflow/models/ExecutionBasicResponse.java

```diff
@@ -6,28 +6,35 @@
 
 
 public class ExecutionBasicResponse {
-  private int executionId;
+  private String executionId;
   private String submitUser;
   private long submitTime;
-  private String experimentId;
   private String concurrentOption;
   private String failureAction;
```

Comment:
Shouldn't these fields be enums?

Reply:
There was an earlier discussion about what should be the exact strings which appear as part of the output(enum values vs names) so I have kept these as strings for now as that can better incorporate both possibilities.
In case we decide to change this it should be part of a separate PR.

Reply:
There was an earlier discussion about what should be the exact strings which appear as part of the output(enum values vs names) so I have kept these as strings for now as that can better incorporate both possibilities.
In case we decide to change this it should be part of a separate PR.

Reply:
In the code I think the should be enums to maintain consistency with the rest of the classes and because of the benefits of dealing with enums vs strings. This works well with whatever way we decided to use either the exact enum value or a string associated with it, what changes is the method we use to _stringify_ them (`toString() or getName()`)

Reply:
Thanks for pointing out. I have made a note of it and updating it as part of an upcoming PR.